### ISSUE TITLE
Make the forum and other community-related links more visibles

### ISSUE THREAD

**Issue by luizbills** [4ian/GDevelop-website#71]
My suggestion:
![image](https://user-images.githubusercontent.com/1798830/68973925-390c8280-07ce-11ea-8c7f-b9b16833fab3.png)

**Comment by 4ian** [comment]
Yes, that would be interesting - though we have to think how this would be displayed on a mobile. 

Any help to implement this is welcome!

**Comment by luizbills** [comment]
> Yes, that would be interesting - though we have to think how this would be displayed on a mobile.

I put a mobile suggestion

> Any help to implement this is welcome!

CSS (and React) scares me

**Comment by Kushagraw12** [comment]
Hey!, Can I take up this issue?

**Comment by 4ian** [comment]
Sure, feel free to take a look! :)

**Comment by Kushagraw12** [comment]
@4ian  I am having some trouble setting up my environment. Can you help me out?
I tried doing ``` npm install``` and ```npm run develop```. they throw me some errors, and I cant get my way around them.

**Comment by 4ian** [comment]
@Kushagraw12 Oh right sorry about this, seems like the dependencies are now a bit old and not working well with Node.js 10/12 😬 You would either need to upgrade them because that's surely tedious to do, or try with Node.js 8 (use something like nvm to have multiple versions of Node.js on your computer). I'll take a look at if I can fix that.

**Comment by Kushagraw12** [comment]
Okay, I have nvm installed. I will just downgrade my node to v8, and let you know if it works!

**Comment by Kushagraw12** [comment]
@4ian I downgraded my node to ```v8.0.0``` and ran ```npm install``` and ```npm run develop```. They still throw me some error and I am unable to set it up locally. I also tried installing gatsby v2.0.0 and the gatsby cli, but the ```npm run develop``` still throws errors. Please help.
![Screenshot (43)](https://user-images.githubusercontent.com/53997603/84876265-a8b85800-b0a4-11ea-8941-22899109f1e5.png)

**Comment by 4ian** [comment]
> v8.0.0

Upgrade to the latest version in the node.js v8 series, i.e: Node.js v8.17.

> They still throw me some error and I am unable to set it up locally

Please paste the errors here that you have after installing the latest Node.js 8 version, removing your node_modules, re-running npm install and then npm run develop.

> I also tried installing gatsby v2.0.0 and the gatsby cli,

This risks mixing the versions of Gatsby, I would not recommend this at all. Migrating to Gatsby v2 needs some changes in the code: https://www.gatsbyjs.org/docs/migrating-from-v1-to-v2/
I think Gatsby 2 and gatsby-cli is not supporting node 8 anymore. In any case, uninstall them entirely, trash your node_modules folder, ensure you have the latest Node 8 and paste the error messages.

**Comment by Kushagraw12** [comment]
Ohkay @4ian , let me try this, and I will let you know the outcome ASAP. 
Thanks!

**Comment by 4ian** [comment]
Sorry about the bad experience, it's a shame that dependencies got incompatible with latest node versions 😬

**Comment by Kushagraw12** [comment]
I did as you told me to, and this is the error that I encountered:
![Screenshot (44)](https://user-images.githubusercontent.com/53997603/84942081-f31af280-b0ff-11ea-89ab-12f3a285b05d.png)
Please help me

**Comment by 4ian** [comment]
That's very strange. Can you confirm if you have files called gatsby/gatsby.cmd/gatsby.ps1 in `node_modules/.bin` folder?

Maybe @Bouh or @arthuro555 could try with Node 8 too on Windows to see if you can run this?

**Comment by Bouh** [comment]
I've already tried node v8.15.0
`npm install `and `npm run develop`

![image](https://user-images.githubusercontent.com/1670670/84956745-2064a200-b0fa-11ea-8a29-e30cfafca4ed.png)

**Comment by Bouh** [comment]
The error more complete when i use `npm run build`

![image](https://user-images.githubusercontent.com/1670670/84958044-c74a3d80-b0fc-11ea-9213-d63199c9b968.png)

**Comment by Kushagraw12** [comment]
> That's very strange. Can you confirm if you have files called gatsby/gatsby.cmd/gatsby.ps1 in `node_modules/.bin` folder?

@4ian  Sorry for the late reply. There was no gatsby file in the `node_modules/.bin` folder.
However, I did found some gatsby files in the `node_modules` folder:
![Screenshot (46)](https://user-images.githubusercontent.com/53997603/85025316-ded50500-b194-11ea-8594-e1be49836b6d.png)
